cmd/kubelet/app/server.go

```diff
@@ -130,6 +132,7 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.MasterServiceNamespace, "master_service_namespace", s.MasterServiceNamespace, "The namespace from which the kubernetes master services should be injected into pods")
 	fs.Var(&s.ClusterDNS, "cluster_dns", "IP address for a cluster DNS server.  If set, kubelet will configure all containers to use this for DNS resolution in addition to the host's DNS servers")
 	fs.BoolVar(&s.ReallyCrashForTesting, "really_crash_for_testing", s.ReallyCrashForTesting, "If true, crash with panics more often.")
+	fs.BoolVar(&s.MountTmpfs, "mount_tmpfs", s.MountTmpfs, "If true, kubelet will mount tmpfs storage")
```

Comment:
Mount tmpfs storage sounds slightly vague. Can we clarify what the storage is for?